### DISCUSSION TITLE
playing 2 audio file simultaneously with each having its own separate volume,

### DISCUSSION THREAD

**Discussion by ashermatsh** [alexmercerind/dart_vlc#257] [General]
I am creating a flutter app, I need to play more than one audio file simultaneously

I created a widget which contains a player Object,and a seek slider which controls the volume of the player.
when I add more than one of the above widgets into the homepage,both them are playing as intended but when I change the volume of  one widget the volume of the other player also gets changed.

**Comment by alexmercerind** [comment]
#90 not a `dart_vlc` bug.

**Comment by ashermatsh** [comment]
I tried the solution given in #90 and it worked , Thank you
